src/libtexture/imagecache.cpp

```diff
@@ -3442,7 +3444,13 @@ ImageCacheImpl::resolve_udim(ImageCacheFile* udimfile, Perthread* thread_info,
                                     Strutil::sprintf("u%d", utile + 1), true);
         realname         = Strutil::replace(realname, "<V>",
                                     Strutil::sprintf("v%d", vtile + 1), true);
-        realfile         = find_file(realname, thread_info);
+        realname
+            = Strutil::replace(realname, "_u##v##",
```

Comment:
man clang-format makes some ugly choices..... ;-)

Reply:
Occasionally it's awful. The small number of places where it looks bad is more than made up for by never having any arguments about how formatting should be done, nor having to explain to anybody more than "make it conform to clang-format." It's a net win for productivity.